docs/update/主数据维护工作流.md

```
@@ -126,7 +126,7 @@ flowchart TD
    - 仓库映射：`item_to_warehouse_dict`
    - 影拓丰碑敌人：`permanent_dict`
 3. 如有高阶关卡，再补 `higher_order_feature_dict`（在 `src/data/world_map.py` 加载器内，因含 FeatureList 枚举）
-4. `scripts/sync_world_map_langs.py` 会自动为主数据新增的名称补齐 `assets/lang/world_map.json` 节点（官方有译名则填，无则至少 zh_CN）；官方新增名称会打印 MANUAL 提示人工决定是否加入主数据
+ 4. `scripts/i18n/sync_world_map_langs.py` 会自动为主数据新增的名称补齐 `assets/lang/world_map.json` 节点（官方有译名则填，无则至少 zh_CN）；官方新增名称会打印 MANUAL 提示人工决定是否加入主数据
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)